### PR TITLE
cluster-ui: insights bug fixes

### PR DESCRIPTION
**Pull request by xinhaoz**
Previously, when the insights statement was refactored to include all stmts for txn insights, the sorting by most recent txn was missed, causing the first execution to be surfaced instead of the most recent. This commit reverses the sorting s.t. we surface the most recent execution of a txn in the insights pages.

Epic: None

Release note: None

### CODE REVIEW

**Review by ericharmeling — APPROVED** [review]
LGTM! Thanks for fixing these bugs!

File: pkg/ui/workspaces/cluster-ui/src/insights/utils.ts [hunk 1]
```diff
@@ -460,7 +460,7 @@ export function getStmtInsightRecommendations(
     indexRecommendations: insightDetails.indexRecommendations,
     databaseName: insightDetails.databaseName,
     elapsedTimeMillis: insightDetails.elapsedTimeMillis,
-    contentionTime: insightDetails.totalContentionTime.asMilliseconds(),
+    contentionTime: insightDetails.totalContentionTime?.asMilliseconds(),
```
Comment (maryliag) [inline pkg/ui/workspaces/cluster-ui/src/insights/utils.ts:463]: this was just fixed on a PR just merged: https://github.com/cockroachdb/cockroach/pull/92571/files#diff-3805bc3aeac419ce79c8b16f1b6dd247608f9a30ec3bfa3e6007cb3350b8eeb7R463

Comment (xinhaoz) [reply pkg/ui/workspaces/cluster-ui/src/insights/utils.ts]: Nice, I'll remove that from the commit description!

**Pull request merged**